steering/elections/2025/README.md

```diff
@@ -0,0 +1,82 @@
+| February 24        | Election begins via Elekto UI                                       |
+| March 9            | Election closes at 2359 UTC (5pm PDT)                               |
+| week of March 10   | Winners announced                                                   |
+
+## Candidacy process
+
+Eligibility for candidacy is defined in the [Steering Committee charter](../../CHARTER.md) as a [project member](../../../ROLES.md#member) who does not work for a Company that will hold a Contribution Seat during the upcoming term.
+
+Members from **Solo.io**, **Microsoft**, **Huawei**, **Google** and **Tetrate** are therefore **ineligible** to stand in this election.
+
+If you want to stand for election, open a PR against the [istio/community repository](https://github.com/istio/community) to include
+your candidate profile in the `/steering/elections/2025` folder, with the file named in the format `candidate-yourname.md`. We have included [a template file](nomination-template.md) as an example. This profile should include:
+
+- Your name
+- Your GitHub ID
+- Your company affiliation (employer or otherwise)
+- Your contributions to Istio
+- Why you are running
+
+Once you have created the PR, you may encourage endorsements as comments on it. After a candidate has met all election requirements, the Election Officers will merge the profile PR.
+
+If you want to nominate someone else, you may do so, but please confirm their interest first.
+
+## Voting process
+
+Eligibility to vote is defined in the [Steering Committee charter](../../CHARTER.md) as either:
+
+- a project member who has had at least one Pull Request merged in the past 12 months, or
+- someone who has submitted the [voting exception form](https://forms.gle/WRgXEwdqR166eghTA) and has been accepted by the steering committee as having standing in the community.
+
+Elections will be held using [Elekto](https://elekto.dev/), an online voting tool
+created for the CNCF. Elekto has one critical advantage over CIVS, which is that
+it uses GitHub OAuth to log you in to vote, instead of relying on email.
+
+Thus, when you go to Elekto, you will be prompted to log in your GitHub account.
+You will then be able to click on "Explore Election" to look at the list of
+elections, and select the "2025 Istio Steering Committee Election."
+
+The election page will, among other things, confirm if you are eligible to vote,
+via a button at the top of the screen.
+
+As candidates file their candidate statements in the community repo, they will
+become visible in the Elekto UI.  You may click through to any candidate to see
+their profile.
+
+Once the vote begins, you will be able to rank the candidates in the order of
+your preference, and submit your ballot.  When you submit, you will be offered
+a chance to set a password, which is required if you want the ability to return
+and re-cast your ballot before August 13th.
```

Comment:
Should this be March 9th?

Reply:
Yes, thank you; there's always one!  I searched for "2023" but not "August"...